automation/functions/inbound.py

```diff
@@ -1,6 +1,2 @@
-class NewCallback:
```

Comment:
I reworked polling dispatch so all the functions modules can be simple python functions again, no more confusing classes near the logic.